src/connector/src/source/google_pubsub/mod.rs

```diff
@@ -25,15 +28,19 @@ pub use source::*;
 pub use split::*;
 use with_options::WithOptions;
 
+use crate::error::ConnectorResult;
 use crate::source::SourceProperties;
 
 pub const GOOGLE_PUBSUB_CONNECTOR: &str = "google_pubsub";
 
 #[derive(Clone, Debug, Deserialize, WithOptions)]
 pub struct PubsubProperties {
-    /// pubsub subscription to consume messages from
-    /// The subscription should be configured with the `retain-on-ack` property to enable
-    /// message recovery within risingwave.
+    /// Pub/Sub subscription to consume messages from.
+    ///
+    /// Note that we rely on Pub/Sub to load-balance messages between all Readers pulling from
+    /// the same subscription. So one `subscription` (i.e., one `Source`) can only used for one MV.
```

Comment:
suggest: one subscription can be shared across one fragment to load-balance the parallelism.